var/spack/repos/builtin/packages/revocap-refiner/package.py

```diff
@@ -28,18 +26,15 @@ class RevocapRefiner(MakefilePackage):
     patch('delete_getIndices.patch')
 
     def edit(self, spec, prefix):
-        cflags = ['-O']
-        cxxflags = ['-O',  self.compiler.cxx_pic_flag]
+        cflags = ['-O2']
+        cxxflags = ['-O2', self.compiler.cxx_pic_flag]
         fflags = ['']
         ldshare = ['']
-        libs = ['-lstdc++']
-        if spec.satisfies('%gcc'):
-            ldshare.append('g++ -shared -s')
-
+        libs = ['']
         m = FileFilter('MakefileConfig.in')
-        m.filter(r'CC\s=.*$', 'CC={0}'.format(spec['mpi'].mpicc))
-        m.filter(r'CFLAGS\s=.*$', 'CFLAGS={0}'.format(' '.join(cflags)))
-        m.filter(r'CXX\s*=.*$',  'CXX={0}'.format(spec['mpi'].mpicxx))
+        m.filter(r'CC\s*=.*$', 'CC={0}')
+        m.filter(r'CFLAGS\s*=.*$', 'CFLAGS={0}'.format(' '.join(cflags)))
+        m.filter(r'CXX\s*=.*$', 'CXX={0}')
```

Comment:
I think there's a typo. Do you really want the string `CC={0}` in your Makefile without the placeholder being formatted?

Reply:
Thanks for pointing that out. I fixed it.